.vib/redis/goss/redis.yaml

```diff
@@ -0,0 +1,15 @@
+file:
+  /opt/bitnami/redis/etc/redis.conf:
+    exists: true
+    mode: "0664"
+    contains:
+      - port {{ .Vars.port }}
```

Comment:
Do not use a variable here. You can replace it by the port instead, although I would personally drop this line as there is no real need of checking every modification in the file